internal/tool/tool.go

```diff
@@ -394,6 +393,14 @@ func parseOutput(toolDefinition codacy.ToolDefinition, commandOutput string) ([]
 	return result, nil
 }
 
+func writeMessage(s string) string {
+	// If message is empty, write a default message
+	if s == "" {
+		return "Potential security issue detected. No specific details available. Please review the identified code segment for potential security vulnerabilities."
```

Comment:
nice!

if we have the ToolDefinition in scope, we could use the Pattern.Title instead
I think it's what Semgrep uses, but even if it isn't it will give users a better idea of what the error is than a static message